extensions/positron-zed/src/positronZedLanguageRuntime.ts

```diff
@@ -1592,20 +1594,25 @@ export class PositronZedRuntimeSession implements positron.LanguageRuntimeSessio
 	 * @param code The Zed code the user entered.
 	 * @param languageId The language identifier
 	 * @param codeToExecute The code to execute.
+	 * @param mode The execution mode to conform to.
 	 */
 	private async simulateCodeExecution(parentId: string,
 		code: string,
 		languageId: string,
-		codeToExecute: string) {
+		codeToExecute: string,
+		mode: positron.RuntimeCodeExecutionMode) {
 		// Enter busy state and output the code.
 		this.simulateBusyState(parentId);
 		this.simulateInputMessage(parentId, code);
 
 		// Let the user know what we're about to do
 		this.simulateOutputMessage(parentId, `Executing ${languageId} snippet: ${codeToExecute}`);
 
+		// Don't focus the console if code should being executed silently
+		const focus = mode !== positron.RuntimeCodeExecutionMode.Silent;
```

Comment:
I figured it makes sense to update the focus value based off the `RuntimeCodeExecutionMode`. Since this is all used for dev testing it doesn't really matter but this felt more sane than always having focus set to true.